examples-native/crna-kitchen-sink/package.json

```diff
@@ -34,18 +36,22 @@
     "@storybook/client-logger": "file:../../packs/storybook-client-logger.tgz",
     "@storybook/components": "file:../../packs/storybook-components.tgz",
     "@storybook/router": "file:../../packs/storybook-router.tgz",
-    "@storybook/core": "file:../../packs/storybook-core.tgz",
+    "@storybook/core": "file:../../lib/core",
     "@storybook/core-events": "file:../../packs/storybook-core-events.tgz",
     "@storybook/node-logger": "file:../../packs/storybook-node-logger.tgz",
     "@storybook/react-native": "file:../../packs/storybook-react-native.tgz",
+    "@storybook/react-native-server": "file:../../app/react-native-server",
```

Comment:
Why not `.tgz`? Can we remove the other `.tgz` files and remove the `build-packs` step entirely?